frontend/packages/dev-console/src/utils/pipeline-augment.ts

```diff
@@ -278,12 +278,14 @@ export const getTaskStatus = (pipelinerun: PipelineRun, pipeline: Pipeline): Tas
         taskStatus[runStatus.Running]++;
       } else if (status === 'Failed') {
         taskStatus[runStatus.Failed]++;
+      } else if (status === 'Cancelled') {
+        taskStatus[runStatus.Cancelled]++;
       } else {
         taskStatus[runStatus.Pending]++;
       }
     });
-    taskStatus[runStatus.Failed] > 0
-      ? (taskStatus[runStatus.Cancelled] =
+    taskStatus[runStatus.Failed] > 0 || taskStatus[runStatus.Cancelled] > 0
+      ? (taskStatus[runStatus.Cancelled] +=
```

Comment:
There are existing tests and I've added a few more. This was mainly because of the `||` the existing tests din't fail. It just added the area for the function. Also, the existing tests were considering `Cancelled+Failed` and not `Cancelled` and `Failed` individually as it wasn't the case before.